.github/workflows/build-artifact-payg-multivm.yaml

```diff
@@ -25,6 +24,14 @@ jobs:
     preflight:
         runs-on: ubuntu-latest
         steps:
+            - name: Get versions of external dependencies
+              run: |
+                curl -Lo external-deps-versions.properties https://raw.githubusercontent.com/Azure/azure-javaee-iaas/main/external-deps-versions.properties
+                source external-deps-versions.properties
+                echo "azCliVersion=${AZ_CLI_VERSION}" >> $GITHUB_ENV
+                echo "bicepVersion=${BICEP_VERSION}" >> $GITHUB_ENV
+                echo "refArmttk=${ARM_TTK_REFERENCE}" >> $GITHUB_ENV
+                echo "refJavaee=${AZURE_JAVAEE_IAAS_REFERENCE}" >> $GITHUB_ENV
```

Comment:
This should have been a separate PR.  @zhengchang907 , in the future, please do not mix unrelated features in the same PR.